### PR TITLE
Docker docs

### PR DESCRIPTION
**Pull request by nvasilakis**
This PR adds documentation on Docker images. This documentation is reflected in PaSh's [Docker hub page](https://hub.docker.com/r/binpash/pash).

### CODE REVIEW

File: scripts/dockerfiles/README.md [hunk 1]
```diff
@@ -1,4 +1,92 @@
...
+```sh
+$ docker build -t app . && docker run -it --init -p 1993:1993 app
+```
+
+## (optional) Add `pash-docker` as an alias to your shell
+
+You can add `pash` command to your shell init file (e.g.`.bashrc`) to launch scripts in the Docker environment.
+
+```sh
+pash () {
```
Comment (angelhof) [inline scripts/dockerfiles/README.md:65]: cool!

File: scripts/dockerfiles/README.md [hunk 2]
```diff
@@ -1,4 +1,92 @@
...
+
+## From PaSh's Official Dockerfiles
+
+```sh
+git clone git@github.com:binpash/pash.git
+cd pash/scripts/docker
+docker build -t "pash:latest" .
+```
+
+## As a Dockerfile
```
Comment (angelhof) [inline scripts/dockerfiles/README.md:29]: Do we need to duplicate the Dockerfile here? What if it changes? We could just point to it :)

**Review by angelhof — APPROVED** [review]
LGTM modulo the minor dockerfile duplication comment

File: scripts/dockerfiles/README.md (hunk 2 above)
Comment (nvasilakis) [reply scripts/dockerfiles/README.md:29]: @angelhof Great idea!

**Pull request merged**